### PR TITLE
Auth page finalization

### PR DESCRIPTION
**Pull request by nastiaymlnv**

### CODE REVIEW

File: src/components/AuthForm/AuthForm.jsx [hunk 1]
```diff
@@ -26,12 +28,14 @@ const formData = {email: null, password: null};
 const AuthForm = () => {
     const store = useStore();
```
Comment (vladhybalo) [inline src/components/AuthForm/AuthForm.jsx]: Looks like you dont use store anymore

File: src/components/AuthForm/AuthForm.jsx [hunk 2]
```diff
@@ -84,75 +88,85 @@ const AuthForm = () => {
...
-            });
         }
         else {
             setValidMail(false);
             setValidPassword(false);
         }
     }
 
+    const authByGoogle = () => {
+        console.log('enter with google');
```
Comment (vladhybalo) [inline src/components/AuthForm/AuthForm.jsx]: pls dont push console logs

File: src/components/AuthForm/AuthForm.jsx [hunk 3]
```diff
@@ -84,75 +88,85 @@ const AuthForm = () => {
...
+        dispatch(fetchGoogleData());
+    }
+
+    useEffect(() => {
+        setFetchLoading(false);
+        if (userInfo.hasOwnProperty('userData')) {
```
Comment (vladhybalo) [inline src/components/AuthForm/AuthForm.jsx]: Why not simply `userInfo.userData` ?

File: src/ducks/userInfo/userInfoActions.jsx [hunk 4]
```diff
@@ -27,6 +43,7 @@ export const fetchSignInData = (formData) => {
             })
             .catch(error => {
                 alert(error.response.data.message);
+                dispatch(errorCredentials(error.response));
```
Comment (vladhybalo) [inline src/ducks/userInfo/userInfoActions.jsx]: As I see, every error will be handled here. Why do you name it error**Credentials**?

File: src/ducks/userInfo/userInfoActions.jsx [hunk 5]
```diff
@@ -37,10 +54,30 @@ export const fetchSignUpData = (formData) => {
...
+        axios
+            .get(`https://kidslike-v1-backend.goit.global/auth/google`, {
+                headers: {
+                    'Access-Control-Allow-Origin' : '*'
+                }
+            })
+            .then(res => {
+                alert('Successful signing in');
+                console.log(res);
+                // dispatch(userGoogleSignIn(res.data));
```
Comment (vladhybalo) [inline src/ducks/userInfo/userInfoActions.jsx]: uncomment this line pls

File: src/ducks/userInfo/userInfoActions.jsx [hunk 6]
```diff
@@ -2,6 +2,8 @@ import axios from 'axios';
 
 export const USER_SIGN_UP = "USER_SIGN_UP";
 export const USER_SIGN_IN = "USER_SIGN_IN";
+export const AUTH_ERROR = "AUTH_ERROR";
```
Comment (vladhybalo) [inline src/ducks/userInfo/userInfoActions.jsx]: nitpick: I would name this action name same as others - starting from `USER_`.
I could be something like user_request_error

**Review by DmytroP888 — DISMISSED** [review]

File: src/ducks/userInfo/userInfoActions.jsx [hunk 7]
```diff
@@ -37,10 +54,30 @@ export const fetchSignUpData = (formData) => {
...
+    return dispatch => {
+        axios
+            .get(`https://kidslike-v1-backend.goit.global/auth/google`, {
+                headers: {
+                    'Access-Control-Allow-Origin' : '*'
+                }
+            })
+            .then(res => {
+                alert('Successful signing in');
+                console.log(res);
```
Comment (vladhybalo) [inline src/ducks/userInfo/userInfoActions.jsx]: to remove

**Review by vladhybalo — APPROVED** [review]

**Review by andriiivaniuk — APPROVED** [review]

**Pull request merged**